src/main/java/nl/hsac/fitnesse/fixture/slim/EmailFixture.java

```diff
@@ -105,25 +110,98 @@ public String ccRecipient() {
         });
     }
 
+    public ArrayList<String> listAttachments() {
+        ArrayList<String> values = null;
+        if(messageAttachments.size() > 0) {
+            values = new ArrayList<>();
+            for (ImapAttachment ia : messageAttachments) values.add(ia.getFileName());
+        }
+        return values;
+    }
+
+    public String saveAttachmentByNumber(int number)
```

Comment:
Wouldn't `saveAttachmentNumber(` make a better command in the wiki e.g. `|show|save attachment number|1|`?